playground/nextjs/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "rm -rf .next && next dev --port 4011",
+    "dev": "rm -rf .next && next dev --port 3000",
```

Comment:
❓ Why did the port change? I am under the impression we picked 4011 as it's available if the local dev env is running.

Reply:
This is done by mistake, I will remove this from the commits